### DISCUSSION TITLE
Multiple jpg's to one per page of single pdf

### DISCUSSION THREAD

**Discussion by ghacky** [ImageMagick/ImageMagick#2055] [Help]
I'm trying to take multiple jpg files and load them into a pdf file with one image per page.  I've used  convert *.jpg out.pdf  which creates a multiple page pdf, but only the first page is visible.  Any suggestions of what i'm doing wrong.  The output file size seems to be a sum of the individual image sizes, but only the first image displays.

**Comment by snibgo** [comment]
What version of IM, on what platform?

What is the text output from:
`convert *.jpg +write info: out.pdf`
This should list all if the images that will be written to the PDF.

What software are you using to view the PDF? I suggest you use Adobe Reader.

**Comment by ghacky** [comment]
Thanks snibgo, I'm running Imagemagick 7 on windows 7 64bit.  I plan to use it on a 2016 windows server if it works. I'm using adobe reader. I  tried converting each to a pdf then using pdftk, which did what I wanted, but i was trying to avoid using two different programs.  I ran the command you offered but just used magick instead of convert and got this:
\\path\still
001.jpg[0] JPEG 1920x1080 1920x1080+0+0 8-bit sRGB 306361B 0.000u 0:00.043
\\path\still
002.jpg[1] JPEG 1920x1080 1920x1080+0+0 8-bit sRGB 561076B 0.000u 0:00.051
\\path\still
003.jpg[2] JPEG 1920x1080 1920x1080+0+0 8-bit sRGB 559933B 0.000u 0:00.058

**Reply by fmw42** [reply]
For ImageMagick 7, use magick not convert. Does that help?

**Comment by snibgo** [comment]
There seems to be a bug. For example, writing three images to a PDF creates an invalid PDF, as measured by pdfimages and Adobe Reader:

````
f:\web\im>%IMG70917%magick rose: wizard: logo: out2.pdf

f:\web\im>pdfimages -list out2.pdf
page   num  type   width height color comp bpc  enc interp  object ID x-ppi y-ppi size ratio
--------------------------------------------------------------------------------------------
   1     0 image      70    46  rgb     3   8  image  no         8  0    72    72 9688B 100%
   2     1 image     480   640  index   1   8  image  no        22  0    72    72 93.1K  31%
Syntax Error: Kid object (page 3) is wrong type (integer)
````


A developer may comment.
